releases/release-5.1.0.md

```diff
@@ -0,0 +1,329 @@
+| TiKV 配置文件  | [`sink-memory-quota`](/tikv-configuration-file.md#sink-memory-quota)  | 新增  | 设置缓存在内存中的 TiCDC 数据变更事件占用内存的上限。默认值为 512MB。 |
+| TiKV 配置文件  | [`incremental-scan-threads`](/tikv-configuration-file.md#incremental-scan-threads)  | 新增  | 控制增量扫描历史数据任务的线程个数。默认值为 4，代表 4 个线程。  |
+| TiKV 配置文件  | [`incremental-scan-concurrency`](/tikv-configuration-file.md#incremental-scan-concurrency)  | 新增  | 控制增量扫描历史数据任务的最大并发执行个数。默认值为 6，代表最多并发执行 6 个任务。 |
+| TiKV 配置文件  | [`soft-pending-compaction-bytes-limit`](/tikv-configuration-file.md#soft-pending-compaction-bytes-limit)  | 修改  | pending compaction bytes 的软限制，默认值从 "64GB" 修改为 "192GB"。 |
+| TiKV 配置文件  | [`storage.io-rate-limit`](/tikv-configuration-file.md#storageio-rate-limit)  | 新增  | 控制 TiKV 写入的 IO 速率。`storage.io-rate-limit.max-bytes-per-sec` 默认值为 “0MB”。 |
+| TiKV 配置文件  | [`resolved-ts.enable`](/tikv-configuration-file.md#enable)  | 新增  | 为所有 Region leader 维护 `resolved-ts`，默认值为 `true`。 |
+| TiKV 配置文件  | [`resolved-ts.advance-ts-interval`](/tikv-configuration-file.md#advance-ts-interval)  | 新增  | 推进 `resolved-ts` 的间隔，默认为 "1s"，支持动态更改。 |
+| TiKV 配置文件  | [`resolved-ts.scan-lock-pool-size`](/tikv-configuration-file.md#scan-lock-pool-size)  | 新增  | 用于初始化 `resolved-ts` 时扫锁的线程数，默认值为 `2`。 |
+
+### 其他
+
+- 为了提升 TiDB 性能，TiDB 的 Go 编译器版本从 go1.13.7 升级到了 go1.16.4。如果你是 TiDB 的开发者，为了能保证顺利编译，请对应升级你的 Go 编译器版本。
+- 请避免在对使用 TiDB Binlog 的集群进行滚动升级的过程中新创建聚簇索引表。
+- 请避免在 TiDB 滚动升级时执行 `alter table ... modify column` 或 `alter table ... change column`。
+- 当按表构建 TiFlash 副本时，v5.1 版本及后续版本将不再支持设置系统表的副本。在集群升级前，需要清除相关系统表的副本，否则升级到较高版本后将无法再修改系统表的副本设置。
+- 在 TiCDC 的 `cdc cli changefeed` 命令中废弃 `--sort-dir` 参数，用户可在 `cdc server` 命令中设定 `--sort-dir`。[#1795](https://github.com/pingcap/ticdc/pull/1795)
+
+## 新功能
+
+### SQL
+
+- 新增 MySQL 8 中的公共表表达式 (Common Table Expression)，为 TiDB 带来递归或非递归查询层次结构数据的能力，满足了人力资源、制造业、金融市场和教育在内的多种应用领域需要使用树形查询实现业务逻辑的需求。在 TiDB 中，你可以通过 `WITH` 语句使用公共表表达式。
+[用户文档](/sql-statements/sql-statement-with.md)，[#17472](https://github.com/pingcap/tidb/issues/17472)
+
+- 新增 MySQL 8 中的动态权限 (Dynamic Privileges)。动态权限用于限制 `SUPER` 权限，为 TiDB 提供更灵活的权限配置，实现对某些操作更细粒度的控制。例如，你可以使用动态权限来创建一个只能执行 `BACKUP` 和 `RESTORE` 操作的用户帐户。
+
+    支持的动态权限包括：
+
+    - `BACKUP_ADMIN`
+    - `RESTORE_ADMIN`
+    - `ROLE_ADMIN`
+    - `CONNECTION_ADMIN`
+    - `SYSTEM_VARIABLES_ADMIN`
+
+    你也可以使用插件来添加新的权限。若要查看全部的动态权限，请执行 `SHOW PRIVILEGES` 语句。[用户文档](/privilege-management.md)
+
+- 新增安全增强模式 (Security Enhanced Mode) 配置项，用于对 TiDB 管理员进行更细粒度的权限划分。安全增强模式默认关闭，如需开启，请参考[用户文档](/system-variables.md#tidb_enable_enhanced_security)。
+
+- 全面加强列类型的在线变更能力，支持通过 `ALTER TABLE` 语句进行列的在线类型修改, 包括但不限于：
+
+    - 从 `VARCHAR` 转换为 `BIGINT`
+    - `DECIMAL` 精度修改
+    - 从 `VARCHAR(10)` 到 `VARCHAR(5)` 的长度压缩
+
+    [用户文档](/sql-statements/sql-statement-modify-column.md)
+
+- 引入新的语法 `AS OF TIMESTAMP`，支持通过 Stale Read 功能从指定的时间点或时间范围内读取历史数据。
+
+    [用户文档](/as-of-timestamp.md)，[#21094](https://github.com/pingcap/tidb/issues/21094)
```

Comment:
`/as-of-timestamp.md` is the document about the usage, perhaps `stale-read.md` is better for introduce this feature.

Reply:
```suggestion
    [用户文档](/stale-read.md)，[#21094](https://github.com/pingcap/tidb/issues/21094)
```